src/components/utils/__integrations__/onfidoApi/configurations.integration.js

```diff
@@ -0,0 +1,66 @@
+import { getTestJwtToken } from '../helpers'
+import { getSdkConfiguration } from '../../onfidoApi'
+import { API_URL } from '../helpers/testUrls'
+import {
+  ASSERT_EXPIRED_JWT_ERROR_OBJECT,
+  EXPIRED_JWT_TOKEN,
+} from '../helpers/mockExpiredJwtAndResponse'
+
+let jwtToken = null
+
+describe('API configurations endpoint', () => {
+  beforeEach(async () => {
+    jwtToken = await getTestJwtToken()
+  })
+
+  test('getSdkConfiguration returns sdk features', async () => {
+    process.env.SDK_SOURCE = 'onfido_web_sdk'
+    process.env.SDK_VERSION = '8.3.0'
```

Comment:
Any reason we want 8.3.0 instead of the current version the script is executed in (as env var or from package.json)?

Reply:
Hey Danny!
Let's imagine the configuration changes on the next version (e.g., 8.4.0). The test would start failing. Thus, we want test data to be static ☺️